utils/cliutils/commandsflags.go

```diff
@@ -2016,6 +2022,10 @@ var commandFlags = map[string][]string{
 		platformUrl, user, password, accessToken, serverId, deleteQuiet, lcDryRun, DistRules, site, city, countryCodes,
 		lcSync, maxWaitMinutes, lcProject,
 	},
+	ReleaseBundleExport: {
+		platformUrl, user, password, accessToken, serverId, lcPathMappingTarget, lcPathMappingPattern, Project,
+		rbRepo, downloadMinSplit, downloadSplitCount, TargetDirectory,
```

Comment:
```suggestion
	downloadMinSplit, downloadSplitCount, TargetDirectory,
```
`rbRepo` isn't relevant to rbv2